TTS/tts/utils/speakers.py

```diff
@@ -349,7 +400,7 @@ def get_speaker_manager(c: Coqpit, data: List = None, restore_path: str = None,
                             "You must copy the file speakers.json to restore_path, or set a valid file in CONFIG.d_vector_file"
                         )
                     speaker_manager.load_d_vectors_file(c.d_vector_file)
-                speaker_manager.set_d_vectors_from_file(speakers_file)
```

Comment:
get_speaker_manager is gonna be removed. User should create its own object as in the multi-speaker VCTK recipes.

Reply:
ok, I removed the call of this function in TTS/bin/extract_tts_spectrograms.py. For now, it is used in BaseTTS, but if you like I can remove it as well.